QuickFIXn/QuickFix.csproj

```diff
@@ -2,16 +2,32 @@
 
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0</TargetFrameworks>
-    <Title>QuickFIX/n</Title>
+    <Title>QuickFIX/n Core</Title>
```

Comment:
Since this is a .NET Standard package, I don’t think it’s appropriate to add “Core” explicitly in the title, as .NET Standard allows usage in all flavors of .NET

Reply:
Hmm, I wasn't intending to suggest that this was a .NET Core library. No, instead, I was trying to make it clear that this contained the "core" logic for creating an application for implementing the FIX protocol using QuickFix\\n, that it's the Core library you need, where as the other, protocol-specific libraries--well, I think that's fairly straightforward. 😉

Reply:
How about “Base” then to avoid confusion

Reply:
That sounds great. I'll have that change up in a few minutes.